src/domains/sign/env/prod/terraform.tfvars

```diff
@@ -3,6 +3,18 @@ env_short = "p"
 domain    = "sign"
 location  = "westeurope"
 
+io_sign_issuer_func = {
+  version  = "io-func-sign-issuer-85f3331e"
```

Comment:
what's this code here?

Reply:
The sha of the merge commit on main we want to release. An ADR explaining the whole process will follow ASAP